client/lib/domains/types.ts

```diff
@@ -22,8 +22,11 @@ export type EmailCost = {
 export type GDPRConsentStatus = keyof typeof gdprConsentStatus | null;
 
 export type GoogleEmailSubscription = EmailSubscription & {
+	expiryDate?: string;
 	pendingTosAcceptance?: boolean;
 	productSlug?: string;
+	purchaseCostPerMailbox?: EmailCost | null;
+	renewalCostPerMailbox?: EmailCost | null;
```

Comment:
I think these properties should be moved to `EmailSubscription` and removed from `TitanEmailSubscription`, as they should not be defined on both subscription types.